File: Benchmark/outros/fibonacci.js
function fibonacci(n) {
    let a = 0;
    let b = 1;
    let result = 0;
    for (let i = 2; i <= n; i++) {
        result = a + b;
        a = b;
        b = result;
    }
    return result;
}

console.time("JS");
// console.log(fibonacci(100000));
console.log(fibonacci(10000));
console.timeEnd("JS");

File: Benchmark/outros/fibonacci.cpp
#include <iostream>
using namespace std;

int fibonacci(int n)
{
    int a = 0;
    int b = 1;
    int result = 0;
    for (int i = 2; i <= n; i++)
    {
        result = a + b;
        a = b;
        b = result;
    }
    return result;
}

int main()
{
    cout << fibonacci(10000);
}